stdlib/public/SwiftShims/swift/shims/CoreFoundationShims.h

```diff
@@ -30,9 +30,24 @@ extern "C" {
 #if __LLP64__
 typedef unsigned long long _swift_shims_CFHashCode;
 typedef signed long long _swift_shims_CFIndex;
+
 #else
 typedef unsigned long _swift_shims_CFHashCode;
 typedef signed long _swift_shims_CFIndex;
+
+#endif
+
+#if !FOUNDATION_HELPER_FILE
```

Comment:
FoundationHelpers.mm *does* import the header, but this header *can't* import it, so we need to provide the decl only for non-FoundationHelpers.mm files. I hate this, suggestions welcome.